extern/boostd-data/couchbase/service.go

```diff
@@ -255,24 +260,51 @@ func (s *Store) ListPieces(ctx context.Context) ([]cid.Cid, error) {
 	return s.db.ListPieces(ctx)
 }
 
-func normalizePieceCidError(pieceCid cid.Cid, err error) error {
-	if err == nil {
-		return nil
-	}
-	if isNotFoundErr(err) {
-		return fmt.Errorf("piece %s: %s", pieceCid, types.ErrNotFound)
-	}
-	return err
+//const pieceCheckFrequencyMillis = 60 * 60 * 1000
+const pieceCheckFrequencyMillis = 1000
```

Comment:
Shouldn't this be set to 1 day -- i.e. 3600 * 24 * 1000 ?

Reply:
Yes, we should change these values once we finish testing 👍